pages/common/josm.md

```diff
@@ -0,0 +1,32 @@
+# josm
+
+> Extensible OpenStreetMap editor for Java 8+.
+> More information: <https://josm.openstreetmap.de/>.
+
+- Launch JOSM:
+
+`josm`
+
+- Launch JOSM in maximized mode:
+
+`josm --maximize`
+
+- Launch JOSM and set a specific language:
+
+`josm --language {{de}}`
+
+- Launch JOSM and reset all preferences to default values:
+
+`josm --reset-preferences`
+
+- Launch JOSM and download a specific bounding box:
+
+`josm --download {{minlat,minlon,maxlat,maxlon}}`
```

Comment:
```suggestion
`josm --download {{minlat|minlon|maxlat|maxlon}}`
```

Unaware if this is right

Reply:
No, you actually need all four of them, seperated by a comma.